strawberry_django/auth/queries.py

```diff
@@ -0,0 +1,7 @@
+import strawberry_django
+
+def current_user():
+    @strawberry_django.field
```

Comment:
Shouldn't this use `django_resolver`?

Reply:
The idea is that `current_user` function returns new field which resolves current user.  Following code does the same thing. Would it be better?

```python
def resolve_current_user(info):
    return info.context.request.user

def current_user():
    return strawberry_django.field(resolver=resolve_current_user)
```

Reply:
Cant you do `current_user = strawberry_django.field(resolver=resolve_current_user)` and use that in schema?

Reply:
Many characters to write if you care :)

I thought that it would be nice to provide simple API

```python
@strawberry.type
class Query:
     me: User = auth.current_user()

@strawberry.type
class Mutation:
    login: User = auth.login()
    logout: auth.logout()
```

Instead of more verbose syntax

```python
@strawberry.type
class Query:
     me: User = strawberry_django.field(resolver=auth.resolve_current_user)

@strawberry.type
class Mutation:
    login: User = strawberry.mutation(resolver=auth.resolve_login)
    logout: strawberry.mutation(resolver=auth.resolve_logout)
```

Reply:
I ment
```py
current_user = strawberry_django.field(resolver=resolve_current_user)

@strawberry.type
class Query:
     me: User = current_user
```
Or better:
```py
@strawberry_django.field
def current_user(info):
    return info.context.request.user

@strawberry.type
class Query:
     me: User = current_user
```